src/porepy/models/poromechanics.py

```diff
@@ -70,40 +68,22 @@ def stress(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
 
 
 class EquationsPoromechanics(
-    mass.MassBalanceEquations,
-    momentum.MomentumBalanceEquations,
+    pp.momentum_balance.MomentumBalanceEquations,
+    pp.fluid_mass_balance.TotalMassBalanceEquations,
 ):
     """Combines mass and momentum balance equations."""
 
-    def set_equations(self):
```

Comment:
I changed the order here, because there was some issue in the tests.
One of them had a hardcoded assumption on the order of equations, which was changed in this PR which uses the super framework fully.